packages/flutter/test/material/search_test.dart

```diff
@@ -1047,6 +1068,11 @@ void main() {
     await tester.pumpAndSettle();
     await tester.tap(find.byTooltip('Close'));
     await tester.pumpAndSettle();
+
+    // showSearch without back button
```

Comment:
nit: add period at the end of sentence.

Reply:
yep, of course. i've updated this line. thank you